alpaca/alpaca_test.go

```diff
@@ -372,24 +372,45 @@ func (s *AlpacaTestSuite) TestAlpaca() {
 			return &http.Response{
 				Body: genBody(Order{
 					Qty:         por.Qty,
+					Notional:    por.Notional,
 					Side:        por.Side,
 					TimeInForce: por.TimeInForce,
 					Type:        por.Type,
 				}),
 			}, nil
 		}
 
+		qty := decimal.New(1, 0)
```

Comment:
[minor] no need for this change

Reply:
yup, forgot to clean this up after reverting the changes that made `Qty` a pointer